phobos.gemspec

```diff
@@ -60,7 +60,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'simplecov'
   spec.add_development_dependency 'timecop'
 
-  spec.add_dependency 'activesupport', '>= 3.0.0'
+  spec.add_dependency 'activesupport', '>= 3.0.0', '< 6'
```

Comment:
Why does this have to be < 6?

Reply:
The latest version before the 6 series would be 5.2.3. In 6.0.0 it looks like `try` is no longer available, so things break. We could try and change all calls to `try` to use the safe navigator (but I think this is more general refactoring that's not part of the scope of this PR).

Reply:
Resolved @dorner ?

Reply:
The issue isn't with `try` (that isn't removed), it's due to certain things not being included when loading specs. See here: https://github.com/phobos/phobos/pull/117#discussion_r338227720

Reply:
@cassiomarques this needs to be fixed. I think this is the last thing stopping this from being released!

Reply:
@dorner Sure, will work on it at some point this week and get it fixed. Thanks!

Reply:
@dorner I've rebased my branch and removed the commit that was limiting the `activesupport` version to `< 6`. Let me know if there are any other adjustments needed. Thanks!